docs/plugins/plugin-structure.md

```diff
@@ -149,3 +150,18 @@ Arguments: `message: Object` – object that you sent from `initializeAsync`
 Use this function to receive data back from your `initializeAsync` function. 
 
 Check `initializeAsync` and `onMessage` [example](./examples.md#using-initializeasync-and-onmessage)
+
+### `settings`
+Type: `Object`
+
+This object is used to specify settings that a plugin user can change. Each setting should include a `description` and a `type`. Other keys include:
+  * `description` -  `String`, description of the setting;
+  * `type` - `String`, used to decide element for rendering a setting.
+    * `string`
+    * `number`
+    * `bool`
+    * `option`
+    * `array`
+  * `defaultValue` - `Any`, default value for the setting;
```

Comment:
@BrainMaestro agree with 2, but not with 3. I think it is possible case to select the list of provided options, without ability to add new ones, but at the same time it is possible to choose only one, but with ability to create it by user. What do you think?